File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_One/JAVA/PerfectDivisor.java


import java.io.*;
class PerfectDivisor{
	
	public static void main(String args[])throws IOException{
		
		System.out.println("Enter the number");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int num = Integer.parseInt(br.readLine());


		for(int itr = 2 ; itr<=num/2 ; itr++){

			if( num%itr == 0)
				System.out.printf("%d ",itr);

		}	

	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_One/Cpp/perfectdivisor.cpp
/*Program 5: Write a Program that accepts an integer from user and prints all of
its perfect divisors.
Input: 24
Output: Perfect Divisors of 24 are: 2 3 4 6 8 12

*/

#include<iostream>

int main(){
	
	int num;
	printf("Enter the number ");
	scanf("%d",&num);

	for(int itr = 2 ; itr<=num/2 ; itr++){
		
		if( num%itr == 0)
			printf("%d ",itr);

	}	
}	
